src/security/index.md

```diff
@@ -80,7 +80,19 @@ Any vulnerability reported for flutter websites like
 docs.flutter.dev doesn't require a release and will be
 fixed in the website itself.
 
-Flutter doesn't have a bug bounty program.
+## Bug Bounty programs
+
+Contributing teams are welcome to include Flutter within the scope
+of their bug bounty programs. To have your program listed, please
+contact `security@flutter.dev`.
+
+Google considers Flutter to be in scope for the [Google Open Source
+Software Vulnerability Reward Program][]. For expediency, we ask that
+reporters please contact `security@flutter.dev` before using Google's
+vulnerability reporting flow.
+
+[Google Open Source Software Vulnerability Reward Program]:
```

Comment:
This looks wrong. It should be indented, and a single line, IIUC.

Reply:
updated. thanks for the review.